test/resources/failure/EMI40_RLC_V4.csv

```diff
@@ -0,0 +1 @@
+2023-09-16,no,,Jim,Jeremy,Smith-Smythe,JK883044D,120/SB09591,17.50,yes,,
```

Comment:
please confirm in which test this and below is used

Reply:
ok so it is not needed then

Reply:
Perhaps not strictly _needed_ but I think it's useful to have files like this so the next developer doesn't have to write his/her own.

Reply:
we have files like that in acceptance tests, please remove

Reply:
Done in cff475f